src/sample/tableColumns/tableWithTabs.js

```diff
@@ -0,0 +1,31 @@
+export default [
+    {
+        name: "id",
+    },
+    {
+        name: "name",
+    },
+    {
+        name: "TwoWords",
```

Comment:
our usual field definition should be snake case already, no?

Reply:
Yes, the default field definition is snake case. I just considered a case if there is "camelCase". Please let me know if I have to remove that check.

Reply:
I think that may lead to misuse and makes the naming not uniformed, thoughts? If we need a different mapping, the user can define the label, field etc?

@wildtron @cedricabuso